README.md

```diff
@@ -324,6 +324,7 @@ complicated.
 Subscriptions are a way of receiving data from a source. For example when
 listening for events from a server using `SSE` or `Websockets` for a
 chat app, or when catching keyboard input for a videogame.
+It has a signature of ```(send, done)``` and allow calling actions and/or reducers went the subcribed event is triggered or some data is recived 
```

Comment:
non english native here, so anything that got past a spell check detection is probably not proper english :)

please correct as you (and the language) see fit